src/main/java/ch/njol/skript/log/LogEntry.java

```diff
@@ -160,13 +161,17 @@ public String toFormattedString() {
 			details = OTHER_DETAILS;
 		}
 
+		String from = this.from;
+		if (!Objects.equals(from, ""))
+			from = "§7   " + from + "\n";
```

Comment:
While the field `from` is annotated as nullable, it doesn't appear to be from anywhere in the implementation. The annotation should therefore be removed, and the condition can be replaced with `!from.isEmpty()`